core/src/main/resources/org/pentaho/platform/util/messages/messages.properties

```diff
@@ -71,4 +71,6 @@ LifecyclePhase.FAILED=Failed
 LifecyclePhase.FAILED_DESC=The work item execution has failed
 LifecyclePhase.RESTARTED=Restarted
 LifecyclePhase.RESTARTED_DESC=The work item execution has been restarted
+LifecyclePhase.REDIRECTED=Redirected
```

Comment:
I don't feel strongly about this, so I can remove all the localization, if you think it's appropriate